mne/gui/_coreg_gui.py

```diff
@@ -80,6 +80,25 @@ class CoregModel(HasPrivateTraits):
     mri = Instance(MRIHeadWithFiducialsModel, ())
     hsp = Instance(InstSource, ())
 
+    # this is needed to trigger a filter update when the head shape source
+    # is changed; or is there another way to trigger _inst_points_changed?
+    inst_points = DelegatesTo('hsp')
+
+    # EEG point-related parameters
+    use_eeg_locations = Bool(True, label="Use EEG electrode locations "
+                             "as head shape points")
+    # head shape points relevant to the Model depend on the filter
+    points_filter = Any(desc="Index to select a subset of the head shape "
+                        "points")
+    omitted_info = Any(desc="Dictionary with number of various points omitted")
```

Comment:
Should probably make this a `DictStrInt` (http://code.enthought.com/projects/files/ETS3_API/enthought.traits.trait_types.html) and assign a default value including the keys read by the CoregFrame (`DictStrInt(..., value={'n_total': 0, ...})`, see http://code.enthought.com/projects/files/ETS3_API/enthought.traits.trait_types.Dict.html). Now I get this when launching the GUI (I think for Any the value at initialization is None): 

```
  File "/Users/christian/Code/mne-python/mne/gui/_coreg_gui.py", line 1407, in _get_omitted_string
    if self.model.omitted_info['n_total'] == 0:
TypeError: 'NoneType' object has no attribute '__getitem__'
```